.browserslistrc

```diff
@@ -1,5 +1,4 @@
 last 2 version
 > 2%
-maintained node versions
```

Comment:
Webpack 5 does not currently support multiple target types, so it's either web or node. I was not sure why we had node in here so removed as a test and it worked. If we do need node then we can't update to 5 until they've resolved this problem. More info here https://webpack.js.org/blog/2020-10-10-webpack-5-release/#improved-target-option